cookbook/teaser-provider.rst

```diff
@@ -0,0 +1,121 @@
+Provider for Teaser-Selection
+=============================
+
+`TeaserProvider` are used to load data for Teaser-Selection. It returns an array
+of Teaser instances, identified by an array of ids which will be passed to the
```

Comment:
Shouldn't it be `identified by ids`? I guess you are talking about the array keys here, which are not an array themselves.